addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -447,10 +501,10 @@ def _import_fattura_pa(self, tree, invoice):
                 message_to_log = []
 
                 # Partner (first step to avoid warning 'Warning! You must first select a partner.'). <1.2>
```

Comment:
```suggestion
                # Partner - We use `not incoming` because the partner tag selectors are on the opposite side of the dict `buyer_seller_info` than the company.
```